slidez-core/src/main/java/com/binarystudio/academy/slidez/domain/user/model/User.java

```diff
@@ -1,26 +1,23 @@
 package com.binarystudio.academy.slidez.domain.user.model;
 
+import com.binarystudio.academy.slidez.domain.googlecredentials.GoogleCredentials;
+import com.binarystudio.academy.slidez.domain.presentation.model.Presentation;
+import com.binarystudio.academy.slidez.domain.userprofile.UserProfile;
 import lombok.AllArgsConstructor;
-import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import org.hibernate.annotations.GenericGenerator;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.EnumType;
-import javax.persistence.Enumerated;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
 import java.util.UUID;
 
-@Entity
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
-@Builder
-@Table(name = "users")
+@Entity
+@Table(name = "\"user\"")
```

Comment:
Very strange table name. Remove double quotes.

Reply:
"User" is a reserved word. Removing quotes will end up in a weird and hard to catch bug

Reply:
Try and remove them. After that, use hibernate to generate database using mappings (you'll need to setup hibernate in old way and write up an xml config file). Job will fail on some databases - definitely on Postgres and MySQL